### DISCUSSION TITLE
http mobile request (m.) vs. desktop request (www.)

### DISCUSSION THREAD

**Discussion by amitosg** [spacecowboy/Feeder#450] [Q&A]
Is there a way to enable or configure http(s) requests to be like a desktop, using www prefix? It seems that Feeder accesses are done with a mobile prefix (m.)?

Thanks,
Amit

**Comment by spacecowboy** [comment]
Feeder does nothing.

If you are seeing a `m.` prefix added to a URL unexpectedly, then it's the website that is adding it.

**Reply by amitosg** [reply]
Got it. Thanks. Yes, I'm seeing an 'm' prefix when adding a new URL and hitting the search button. Feeder reports an error for a URL that starts with 'm' while the URL entered in the search line had a 'www' prefix. I thought it had to do with the user agent string used for the request. Thus the ask, if there was a way to override the user agent string or use a desktop request.

**Reply by spacecowboy** [reply]
It's hard to know if overriding the user agent would solve it but it could be. Currently it's not possibly to override the user agent in the UI though. Feel to make a feature request though
